src/main/kotlin/org/team5419/fault/math/pid/PIDF.kt

```diff
@@ -8,6 +8,7 @@ public class PIDF {
     public var kI: Double
     public var kD: Double
     public var kF: Double
+    public var totalMax: Double
```

Comment:
I personally dont thing the PIDF controller itself should cap the output.

Reply:
More elaboration: The subsystem itself should be the one that caps output. The subsystem should be the only class that knows the physical parameters of itself, adding an output cap to the pidf controller effectively defines the max output voltage in an auxiliary class.

Reply:
Isn't the point of this class to increase readability while reducing the need to rewrite code?
I think allowing people to get the PDF do it while making neater and reduce the risk of bugs by having people misemployment it in the future.

Reply:
I think it’s good for now. Let’s merge